src/Symfony/Component/Security/Core/Tests/Authorization/TraceableAccessDecisionManagerTest.php

```diff
@@ -40,4 +41,13 @@ public function provideObjectsAndLogs()
         yield array(array(array('attributes' => array('ATTRIBUTE_1'), 'object' => $x = array(), 'result' => false)), $x);
         yield array(array(array('attributes' => array('ATTRIBUTE_1'), 'object' => $object, 'result' => false)), $object);
     }
+
+    public function testDebugAccessDecisionManagerAliasExistsForBC()
+    {
+        $adm = new TraceableAccessDecisionManager(new AccessDecisionManager());
+
+        if (! $adm instanceof DebugAccessDecisionManager) {
+            $this->fail('Doesn\'t work, BC with SecurityBundle broken, see PR #21088');
```

Comment:
we don't reference github, `git blame` is used instead to look for historical things. I propose:
`$this->fail('For BC, TraceableAccessDecisionManager must be an instance of DebugAccessDecisionManager');`